x-pack/plugins/reporting/common/types/index.ts

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import type { ScreenshotResult } from '../../../screenshotting/server';
```

Comment:
@dokmic we should file an issue (low priority) to move this interface to `common` under the screenshots plugin.

Reply:
That might be problematic since it has `Buffer` in it. That's Node.js's data structure.